exercises/rna-transcription/README.md

```diff
@@ -18,6 +18,8 @@ each nucleotide with its complement:
 * `T` -> `A`
 * `A` -> `U`
 
+Given invalid output, your program should return the first invalid character.
+
```

Comment:
```suggestion


```

Reply:
When you run `bin/ensure-readmes-are-updated.sh` it seems to want this extra linebreak.

Reply:
Ah I was worried that this might happen, thanks.